clients/web/src/lib/components/atoms/Button/AltButton.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+    export let label:string;
+</script>
+
+<button type="button" class="text-white bg-secondary-700 hover:bg-primary-800 
+focus:ring-4 focus:ring-primary-300 font-medium rounded-lg text-sm 
+px-5 py-2.5 mr-2 mb-2 focus:outline-none">Alternative</button>
```

Comment:
Instead of having a `AltButton` component, lets combine it with `Button` and have a `variant` prop that can either be `"primary"` or `"secondary"`, and defaults to `"primary"`

Reply:
Done.